crates/sui-replay/src/lib.rs

```diff
@@ -32,6 +33,12 @@ pub mod types;
 static DEFAULT_SANDBOX_BASE_PATH: &str =
     concat!(env!("CARGO_MANIFEST_DIR"), "/tests/sandbox_snapshots");
 
+pub(crate) const MAINNET_CHAIN_ID: &str = "35834a8a";
+// TODO: Since testnet periodically resets, we need to refresh this value
+// Come up with a better way to refresh this const
+// One option is by reconstructing the genesis
+pub(crate) const TESTNET_CHAIN_ID: &str = "4c78adac";
+
```

Comment:
Why can this not be shared with core protocol? i.e. https://github.com/MystenLabs/sui/blob/c5ae3de27d17f6ae9b7f875f7b3e6966320dc706/crates/sui-types/src/digests.rs#L179-L203

Reply:
Ah I was not aware of this code. Will use it.